### PR TITLE
Adding chaos testing examples within Red Hat/OpenShift

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Start to how chaos testing is being used/tested within Red Hat

### Fixes
https://issues.redhat.com/browse/CHAOS-38

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -198,3 +198,24 @@ Let’s take a look at few recommendations on how and where to run the chaos tes
...
+
+    2. We run the chaos scenarios during a long running reliability test
+
+        i. During this test we perform different types of tasks by different users on the cluster
+
+        ii. We have added the execution of kraken to perform a certain times throughout the long running test and monitor the health of the cluster
+
+        iii. This test can be seen here: https://github.com/openshift/svt/tree/master/reliability-v2
+
+    3. We are starting to add in test cases that perform chaos testing during an upgrade (not many iterations of this have been compelted)
```
Comment (chaitanyaenr) [inline docs/index.md:221]: s/compelted/completed

File: docs/index.md [hunk 2]
```diff
@@ -198,3 +198,24 @@ Let’s take a look at few recommendations on how and where to run the chaos tes
 - Run multiple chaos tests at once to mimic the production outages
   - For example, hogging both IO and Network at the same time instead of running them separately to observe the impact.
   - You might have existing test cases, be it related to Performance, Scalability or QE, run the chaos in the background during the test runs to observe the impact. Signaling feature in Kraken can help with coordinating the chaos runs i.e start, stop, pause the scenarios based on the state of the other test jobs.
+
+
+#### Chaos testing within OpenShift Organization
```
Comment (chaitanyaenr) [inline docs/index.md:203]: @paigerube14 Maybe change the title to Chaos testing in practice within OpenShift Organization? Thoughts?

File: docs/index.md (hunk 1 above)
Comment (paigerube14) [reply docs/index.md]: thanks, fixed!

**Comment by paigerube14** [comment]
There was a spacing issue in [docs/_config.yml](https://github.com/cloud-bulldozer/kraken/pull/204/files#diff-f64371f310bcd3bbf6ead5827d929a4eb13b7516806d6658485a996e39eb2556) that I fixed using "pre-commit run --all"

File: docs/index.md [hunk 3]
```diff
@@ -198,3 +198,24 @@ Let’s take a look at few recommendations on how and where to run the chaos tes
...
+
+    2. We run the chaos scenarios during a long running reliability test
+
+        i. During this test we perform different types of tasks by different users on the cluster
+
+        ii. We have added the execution of kraken to perform a certain times throughout the long running test and monitor the health of the cluster
+
+        iii. This test can be seen here: https://github.com/openshift/svt/tree/master/reliability-v2
+
+    3. We are starting to add in test cases that perform chaos testing during an upgrade (not many iterations of this have been compeleted)
```
Comment (chaitanyaenr) [inline docs/index.md:221]: s/compeleted/completed

**Review by chaitanyaenr — APPROVED** [review]
Minor nit, other than that LGTM!

File: docs/index.md (hunk 3 above)
Comment (paigerube14) [reply docs/index.md]: updated

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

File: docs/index.md [hunk 4]
```diff
@@ -198,3 +198,24 @@ Let’s take a look at few recommendations on how and where to run the chaos tes
...
+
+#### Chaos testing in Practice within the OpenShift Organization
+
+Within the OpenShift organization we use kraken to perform chaos testing throughout a release before the code is available to customers
+
+    1. We execute kraken during our regression test suite
+
+        i. We cover each of the chaos scenarios across different clouds
+
+            a. Our testing is prodominently done on AWS, Azure and GCP
```
Comment (mffiedler) [inline docs/index.md:211]: nit: predominantly

Comment (paigerube14) [reply docs/index.md]: fixed!

**Pull request merged**